view.go

```diff
@@ -275,7 +269,7 @@ func (v *View) newFragment(path string, slice uint64) *Fragment {
 	frag := NewFragment(path, v.index, v.frame, v.name, slice)
 	frag.CacheType = v.cacheType
 	frag.CacheSize = v.cacheSize
-	frag.LogOutput = v.LogOutput
+	frag.Logger = v.Logger
```

Comment:
@tgruben The fragment's logger gets set to the view's logger on fragment creation.

Reply:
Please ignore the crazy man behind the curtain, i was looking at wrong version of view.go.  Nothing to see here, move on :)